ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/snmp-settings.j2

```diff
@@ -102,18 +117,26 @@ snmp-server location {{ snmp_server.location }}
 {%             if user.group is arista.avd.defined %}
 {%                 set user_cli = user_cli ~ " " ~ user.group %}
 {%             endif %}
+{%             if user.remote_address is arista.avd.defined %}
```

Comment:
Remotes only work with v3. Maybe I told you to remove this earlier :) but I thought it was nested inside the v3 check.
```suggestion
{%             if user.remote_address is arista.avd.defined and user.version is arista.avd.defined('v3') %}
```